tests/utils/setup.rs

```diff
@@ -0,0 +1,204 @@
+use fuels::{
+    test_helpers::{launch_custom_provider_and_get_wallets, WalletsConfig},
+    types::ContractId,
+};
+use pyth_sdk::pyth_utils::Pyth;
+
+pub(crate) async fn setup_environment() -> (ContractId, Pyth) {
+    // Launch a local network and deploy the contract
+    let mut wallets = launch_custom_provider_and_get_wallets(
+        WalletsConfig::new(
+            Some(1),             /* Single wallet */
+            Some(1),             /* Single coin (UTXO) */
+            Some(1_000_000_000), /* Amount per coin */
+        ),
+        None,
+        None,
+    )
+    .await
+    .unwrap();
+    let deployer_wallet = wallets.pop().unwrap();
+
+    let pyth = Pyth::deploy(&deployer_wallet).await;
+
+    (pyth.instance.contract_id().into(), pyth)
+}
+
+// const ORACLE_CONTRACT_BINARY_PATH: &str = "./out/debug/pyth-contract.bin";
```

Comment:
Commented code should be removed if deprecated

Reply:
Agree